server/controllers/bookingsController.ts

```diff
@@ -0,0 +1,42 @@
+import type { Booking } from 'approved-premises'
+import type { Request, Response, RequestHandler } from 'express'
+import createDate from '../helpers/createDate'
+
+import BookingService from '../services/bookingService'
+
+export default class BookingsController {
+  constructor(private readonly bookingService: BookingService) {}
+
+  new(): RequestHandler {
+    return (req: Request, res: Response) => {
+      const { premisesId } = req.params
+
+      return res.render(`premises/booking/new`, { premisesId })
+    }
+  }
+
+  create(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { premisesId } = req.params
+      const { CRN, keyWorker }: Booking = req.body
+      const arrivalDay = req.body['arrival-day']
+      const arrivalMonth = req.body['arrival-month']
+      const arrivalYear = req.body['arrival-year']
+      const expectedDepartureDay = req.body['expected-departure-day']
+      const expectedDepartureMonth = req.body['expected-departure-month']
+      const expectedDepartureYear = req.body['expected-departure-year']
+
+      const arrivalDate = createDate(arrivalDay, arrivalMonth, arrivalYear)
+      const expectedDepartureDate = createDate(expectedDepartureDay, expectedDepartureMonth, expectedDepartureYear)
```

Comment:
I've got another way of creating dates in my branch. Would be good if you could take a look at my method and then standardise on one way or another 👍